agir/lib/mailing.py

```diff
@@ -152,6 +152,9 @@ def send_mosaico_email(
         for recipient in recipients:
             # recipient can be either a Person or an email address
             if isinstance(recipient, Person):
+                recipient.ensure_role_exists()
+                if not recipient.role.is_active:
+                    break
```

Comment:
```suggestion
                    continue
```
`break` arrête la boucle complètement, alors que `continue` n'arrête pas la boucle mais passe directement à l’itération suivante